hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheRemoveAllOperation.java

```diff
@@ -120,6 +123,11 @@ public Operation getBackupOperation() {
     }
 
     @Override
+    public ObjectNamespace getServiceNamespace() {
+        return cache != null ? cache.getObjectNamespace() : CacheService.getObjectNamespace(name);
```

Comment:
Optimises not creating the record store if one doesn't exist (same as the `run` method)